### DISCUSSION TITLE
打开一个新的项目时，如何自动切换vs code的工作区

### DISCUSSION THREAD

**Discussion by lihebb** [github0null/eide#387] [Q&A]
![image](https://github.com/user-attachments/assets/ea003258-13cf-4dcd-9b9f-f1d9fd869313)

如图，当我需要切换工程的时候，我会关闭当前工程，然后打开新的工程。但是打开工程会提示是否切换工作区，如果忘记点了，那么vs code还保留着上一个工作区的状态，上一个工作区打开过的文件依然在编辑状态，这样可能出现误操作。

有什么办法规避吗？或者有什么办法自动切换工作区吗？

**Comment by l164908060** [comment]
同求 直接自动继续  不用每次确认要鼠标点一下
